plugins/inputs/reader/README.md

```diff
@@ -0,0 +1,23 @@
+# Reader Input Plugin
+
+The Reader Plugin updates a list of files every interval and parses the data inside.
+Files will always be read from the beginning.  
+This plugin can parse any "data_format" formats.
+
+### Configuration:
+```toml
+[[inputs.reader]]
+## Files to parse each interval.
+## These accept standard unix glob matching rules, but with the addition of
+## ** as a "super asterisk". ie:
+##   /var/log/**.log     -> recursively find all .log files in /var/log
+##   /var/log/*/*.log    -> find all .log files with a parent dir in /var/log
+##   /var/log/apache.log -> only tail the apache log file
```

Comment:
This may cause some slight confusion, `tail` implies that only new additions to the file will be "read", yet line 4 states that "Files will always be read from the beginning."